app/src/main/res/values-ru/strings.xml

```diff
@@ -196,4 +196,7 @@
     <string name="cut">Вырезать</string>
     <string name="map_folders">Карта папок</string>
     <string name="map_folders_summary">Рекурсивное отображение папок для вычисления их размеров (следует отключать при открытии больших томов)</string>
+    <string name="import_folder">Импорт/шифрование папку</string>
+    <string name="import_cancelled">Импорт был отменен</string>
+    <string name="import_folder_in_progress">Импортирование папки (это может занять некоторое время)</string>
```

Comment:
For translations, I used Google Translate, and I'm not sure how readable these are.